### PR TITLE
fix(categoryPath): update categoryPath demension map variable.

### PR DESCRIPTION
**Pull request by sanjithkumar017**
This Pr is obsolete. closing it.

### CODE REVIEW

File: UnbxdSearch_IO.js [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
             return this;
         },
         addFilter: function (field, value) {
-            if (this.options.facetMultilevel && field === (this.options.mappedFields.categoryField || 'categoryPath')) {
+            if (this.options.facetMultilevel && field === (this.options.mappedFields.categoryPath || 'categoryPath')) {
```
Comment (ganeshparsads) [inline UnbxdSearch_IO.js:1076]: why we need this change.

Comment (sanjithkumar017) [reply UnbxdSearch_IO.js:1076]: We have an API for dimension map fields. Which returns object 
`{categoryPath: "ProductCategory"
description: "Description"
imageUrl: "ImageURL"
price: null
productUrl: null
title: "title"}`
So there is a change in the key from `categoryField`->`categoryPath`.

Comment (divyadaglia-unbxd) [reply UnbxdSearch_IO.js:1076]: @sanjithkumar017 
Option 1:
In that case we need to make this change in the sdk also, from mappedfields.categoryfield to mappedfields.categorypath. 
The name was supposed to be generic, hence the terminology categoryfield.
Option 2:
We can make the response change in OnboardingMW.js, and change categoryPath to categoryField, as that is correct terminology.

Comment (ganeshparsads) [reply UnbxdSearch_IO.js:1076]: Yes, It's better to change the response.

**Pull request closed without merging**